app/models/api_service.rb

```diff
@@ -20,7 +21,7 @@ class APIService < ApplicationRecord
   scope :ordered, -> { order(:name) }
 
   def ai_backend
-    openai? ? AIBackend::OpenAI : AIBackend::Anthropic
+    self.url == APIService::URL_GEMINI ? AIBackend::Gemini : (openai? ? AIBackend::OpenAI : AIBackend::Anthropic)
```

Comment:
```suggestion
    case driver
    when :openai
      AIBackend::OpenAI
    when :anthropic
      AIBackend::Anthropic
    when :gemini
      AIBackend::Gemini
    end
```

Reply:
I’m pretty sure `.driver` will return a symbol, but double check that. Also, you need to add :gemini as to to the `enum driver` line above